KtSPR--L4-T3--выращивание бобов

DenisGubernatorov opened the pull request:
Определение являются ли условия подходящими для посадки бобовых

DenisGubernatorov committed:
KtSPR--L4-T3--выращивание бобов
Определение являются ли условия подходящими для посадки бобовых

ievetrovdev commented on src/main/kotlin/lesson4/lesson4_task3.kt:
```diff
@@ -0,0 +1,16 @@
+const val ACCEPTABLE_HUMIDITY = 20
+
+fun main() {
+    var isShiny = true
```
булевые благоприятные значения тоже лучше вынести в константы и с ними сравнивать, так легче масштабировать приложение

DenisGubernatorov committed:
KtSPR--L4-T3--выращивание бобов
1) Добавлены константы допустимых состояний для погоды и тента
2) Текущее состояние погоды и состояние тента переведены на сравнение с допустимыми
